src/Missings.jl

```diff
@@ -490,4 +490,13 @@ function filter(f, itr::SkipMissingsofArrays)
     y
 end
 
+union_macro_ex = :(
+macro _(typ)
+        :(Union{$(esc(typ)), Missing})
+end
+)
+
+union_macro_ex.args[1].args[1] = :?
+eval(union_macro_ex)
```

Comment:
```suggestion
macro var"?"(typ)
    :(Union{$(esc(typ)), Missing})
end
```
if we're cool with requiring Julia 1.3+ (I am at least :stuck_out_tongue:)